oauth2_provider/admin.py

```diff
@@ -19,16 +19,22 @@ class ApplicationAdmin(admin.ModelAdmin):
 class GrantAdmin(admin.ModelAdmin):
     list_display = ("code", "application", "user", "expires")
     raw_id_fields = ("user", )
+    search_fields = ("code", "user__email")
```

Comment:
Hmm... You sure this won't break things it user model doesn't have a email field?

Reply:
+1. Rest looks good.